### ISSUE TITLE
Sawish Oscillator has pronounced DC offset

### ISSUE THREAD

**Issue by Chaircrusher** [AScustomWorks/AS#3]
This simple oscillator + scope patch demonstrates. Not a huge deal in isolation, but when used with other processing modules (i.e. phaser, reverb, anything with a feedback path) the offset is magnified to the point that the waveform is pushed up against 0DbFS and the sound disappears.
[AS-Saw-DCOffset.zip](https://github.com/AScustomWorks/AS/files/1629645/AS-Saw-DCOffset.zip)

**Comment by Chaircrusher** [comment]
These two scopes demonstrate the issue (which btw isn't a problem limited to Sawish, it's all over VCV): The Sawish output is assymetric around 0 with mod at the leftmost setting. It becomes more symmetric as you raise the mod level. But the output min/max of Sawish through reverb shows how feedback processing magnifies the DC offset. The AS Phaser plugin also does this (magify DC offset)
![as-sawish](https://user-images.githubusercontent.com/482621/34919438-a46eb382-f929-11e7-9bfa-c08c3f605346.jpg)

**Comment by AScustomWorks** [comment]
Thanks Kent, I added the dc blocker code from Antonio on FB and tweaked the osc code a bit, so this issue is kind of fixed. Using reverb on the signal now doesn't push it too far. To be honest this module was not meant to be taken too serious(nor the original developer code I think hehe), so it is what it is until I get more proficient with dsp stuff. Implemented code will appear on the next release version (0.5.4)
